src/platform/site-wide/sass/modules/_m-campaign-landing-page.scss

```diff
@@ -0,0 +1,36 @@
+.va-c-story-teaser {
+  img {
+    max-height: 150px;
+  }
+}
+
+.va-c-white-box {
+  width: 220px;
+}
+
+.va-c-blue-line {
+  width: 40px;
+}
+
+.va-c-li-box {
+  min-width: 280px;
+  width: 280px;
+}
+
+.va-c-benefit-category {
+  min-width: 280px;
+}
```

Comment:
Are the various width values here just to help get us started, or do we definitely need them? It would be cool if we could get away without them by replacing them with column-widths using the grid classes

Reply:
ya just to get us started like you said! We can totally omit them moving forward I think! 🙂